recipe_scrapers/usdamyplate.py

```diff
@@ -0,0 +1,72 @@
+from ._abstract import AbstractScraper
+from ._utils import get_minutes, get_yields, normalize_string
+
+
+class USDAMyPlate(AbstractScraper):
+    @classmethod
+    def host(cls):
+        return "myplate.gov"
+
+    def title(self):
+        return self.soup.h1.get_text().strip()
+
+    def total_time(self):
+        # not in every recipe has time given
+        full_detail = self.soup.find(
+            "div", {"class": "mp-recipe-full__overview desktop:grid-col-5 grid-row"}
+        )
+
+        minutes = 0
+        for span in full_detail.findAll(
+            "span", {"class": "mp-recipe-full__detail--data"}
+        ):
+            if "minute" in span.get_text().lower() or "hour" in span.get_text().lower():
+                minutes += get_minutes(span)
+
+        if minutes == 0:
+            return None
+
+        return minutes
+
+    def yields(self):
+        full_detail = self.soup.find(
+            "div", {"class": "mp-recipe-full__overview desktop:grid-col-5 grid-row"}
+        )
+
+        spans = full_detail.findAll("span")
+        i = 0
+        for span in spans:
+            if "Makes:" in span:
+                return get_yields(spans[i + 1])
+            i += 1
+
+        return ""
```

Comment:
Should this be a `return None`?  (or, equivalently, a `return` statement could be omitted here)

(and if not, I'd be curious to hear why -- the method return types and signatures could perhaps do with improvements)